models/public/yolo-v3-tiny-tf/accuracy-check.yml

```diff
@@ -0,0 +1,97 @@
+models:
+  - name: yolo_v3_tiny_tf
+    launchers:
+      - framework: tf
+        model:   yolo-v3-tiny.pb
+        adapter:
+          type: yolo_v3
+          anchors: tiny_yolo_v3
+          num: 6
+          coords: 4
+          classes: 80
+          threshold: 0.001
+          anchor_masks: [[3,4,5],[1,2,3]]
+          raw_output: True
+          output_format: HWB
+          cells: [13,26]
+          outputs:
+            - conv2d_9/BiasAdd
+            - conv2d_12/BiasAdd
+        inputs:
+          - name: 'image_input'
+            type: INPUT
+            shape: [1,416,416,3]
```

Comment:
```suggestion
            shape: [1, 416, 416, 3]
```

Reply:
BTW, if I right remember, current tf launcher implementation does not support shape specification. Do you update it?